src/quo/components/tags/tag.cljs

```diff
@@ -81,39 +81,29 @@
     - `blurred`  boolean: use to determine border color if the background is blurred
     - `type`     can be icon or emoji with or without a tag label
     - `labelled` boolean: is true if tag has label else false"
-  [_ _]
-  (fn
-    [{:keys [id
-             on-press
-             disabled?
-             size
-             active
-             accessibility-label
-             label
-             resource
-             type
-             labelled?
-             blurred?
-             icon-color
-             override-theme]
-      :or   {size 32}}]
-    (let [state (cond disabled? :disabled
-                      active    :active
-                      :else     :default)
-          {:keys [border-color blurred-border-color text-color]}
-          (get-in themes [(or override-theme (theme/get-theme)) state])]
-      [rn/view {:style {:align-items :center}}
-       [base-tag/base-tag
-        {:id                  id
-         :size                size
-         :border-width        1
-         :border-color        (if blurred?
-                                blurred-border-color
-                                border-color)
-         :on-press            on-press
-         :accessibility-label accessibility-label
-         :disabled?           disabled?
-         :type                type
-         :labelled?           (if (= type :label) true labelled?)}
-        [tag-resources size type resource icon-color label text-color labelled?]]])))
+  [{:keys [id on-press disabled? size active accessibility-label label resource type
+           labelled? blurred? icon-color override-theme]
```

Comment:
could you remove use of `override-theme` prop while you're working on this? 🙏

Reply:
Done :+1: